.github/workflows/manage-stale-issues-prs.yml

```diff
@@ -0,0 +1,44 @@
+name: 'Close stale issues and PRs'
+
+env:
+  STALE_AFTER_INACTIVE_DAYS: 90
+  CLOSE_AFTER_INACTIVE_DAYS: 7
+
+on:
+  schedule:
+    - cron: '1 3 * * 0'  # runs every Sunday at 03h01
+
+jobs:
+  stale:
+    permissions:
+      issues: write
+      pull-requests: write
+    if: github.repository_owner == 'geopython'
```

Comment:
The idea was to prevent this from running on forks of this repo (which would thereby be owned by another user/org) - I don't think this would ever run on other `geopython` repos. But I'll change it anyway :)